src/utils/NetworkUtils.ts

```diff
@@ -114,6 +114,15 @@ export function chainIsLinea(chainId: number): boolean {
   return [CHAIN_IDs.LINEA].includes(chainId);
 }
 
+/**
+ * Determines whether a chain ID is a DoctorWho implementation.
+ * @param chainId Chain ID to evaluate.
+ * @returns True if chainId is a DoctorWho chain, otherwise false.
+ */
+export function chainIsDoctorWho(chainId: number): boolean {
+  return [CHAIN_IDs.DOCTOR_WHO].includes(chainId);
+}
```

Comment:
We should probably clean up all these specific on-shot functions (for the future)

Reply:
I'm not really sure what we use these for, or even if they're needed tbh, but it was in the ticket. We only have them for a couple of chains anyway

Reply:
We don't need this :+1: 
```suggestion
```